src/Reports/ElementsInUseReport.php

```diff
@@ -109,6 +115,30 @@ public function getReportField()
         return $field;
     }
 
+    /**
+     * When used with newer versions of silverstripe-reports this method will automatically be added as breadcrumbs
```

Comment:
Can you be more explicit with "newer"? This would have a tendency to confuse me in 2 years time 😆it'd be silverstripe/reports >=4.4 right?